### PR TITLE
Allow drills to have more than one blocked items

### PR DESCRIPTION
**Pull request by Mythril382**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/src/mindustry/world/blocks/production/Drill.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class Drill extends Block{
     /** Speed at which the drill speeds up. */
     public float warmupSpeed = 0.015f;
     /** Special exemption item that this drill can't mine. */
-    public @Nullable Item blockedItem;
+    public Seq<Item> blockedItems = new Seq<>();
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/production/Drill.java:41]: Should be an ObjectSet for fast contains().

**Comment by Anuken** [comment]
> I have ensured that any new features in this PR function correctly in-game, if applicable.

**Comment by Mythril382** [comment]
you could just do this yourself.

**Comment by Mythril382** [comment]
just ignore this weird behavior of mine.....

**Pull request closed without merging**